css/subscribe.css

```diff
@@ -0,0 +1,75 @@
+
+/** BEM CSS USED **/
+
+.subscribe{
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	margin: 50px 0px;
+	width: 100%;
+	padding: 0;
+	text-align: center;
+}
+
+.subscribe__heading{
+	display: flex;
+	flex-direction: column
+}
+
+.subscribe__heading > span{
+	margin: 0px auto 10px;
+	font-size: 1.9rem;
+	font-weight: 500;
+}
+.subscribe__heading > span:nth-of-type(2){
+	font-size: 1.8rem;
+}
+
+.subscribe__inner{
+	margin:15px auto;
+	display: flex;
+	justify-content: center;
+	align-self: center;
+	flex-wrap: wrap;
+}
+.subscribe__input{
+	height: 30px;
+	width: 210px;
+	border: none;
+	font-size: 1rem;
+	margin: 10px 15px;
+	border: 2px solid rgb(227, 4, 100);
+	border-radius: 4px;
+	padding-left: 7px;
+	padding-top: 2px;
+	outline: none;
+}
+
+.subscribe__input:focus,.subscribe__input:active{
```

Comment:
NIT: spacing between the separator and the next class selector
(.subscribe__input:focus, .subscribe__input:active)

Reply:
Space added.